tests/system/common.py

```diff
@@ -780,6 +781,69 @@ def task_by_name(tasks, name):
 
     assert False, "Did not find task with name %s in this list of tasks: %s" % (name, tasks,)
 
+def assert_app_in_all_domains(app, regions=None, zones=None):
+    """Asserts that all tasks of the given app are within all of the given region and/or zone
```

Comment:
Can a task be in several regions? Or do you mean the tasks are distributed across the regions?

Reply:
"All the tasks of the given app are distributed across the regions" let me rephrase it 😄